.travis.yml

```diff
@@ -33,7 +33,8 @@ after_success:
   # to get coverage data with relative paths and not absolute we have to
   # execute coveralls from the base directory of the project,
   # so we need to move the .coverage file here :
-  - if [[ $TEST_SUITE == 'unit' ]]; then mv test/.coverage . && coveralls debug && coveralls -v --rcfile=test/.coveragerc; fi
+  - echo "Python version: $TRAVIS_PYTHON_VERSION"
+  - if [[ $TEST_SUITE == 'unit' ]]; then if [[ $TRAVIS_PYTHON_VERSION == '2.7' ]]; then ./.travis/report_coveralls.sh; fi; fi
```

Comment:
[[ $TEST_SUITE == 'unit' && $TRAVIS_PYTHON_VERSION == '2.7' ]] ?  I guess we have bash in the context